knowledge_graphs/validators/import_validator.py

```
@@ -9,7 +9,7 @@
 
 from knowledge_graphs.ai_script_analyzer import ImportInfo
 from knowledge_graphs.ast_utils import is_standard_library_module, normalize_module_name
-from knowledge_graphs.knowledge_graph_validator_refactored import ImportValidation, ValidationResult, ValidationStatus
+from knowledge_graphs.knowledge_graph_validator import ImportValidation, ValidationResult, ValidationStatus
```

Pylint (3.3.7)

[convention] Line too long (107/100)

(C0301)